Documentation.docc/V4_API_Migration_guide.md

```diff
@@ -134,6 +134,8 @@ These types replace native StoreKit types in all public API methods that used th
 | ------------ | ------------------------------------- | 
 | RCPurchaserInfo | RCCustomerInfo |
 | RCTransaction | RCStoreTransaction |
+| RCTransaction.productId | RCStoreTransaction.productIdentifier |
```

Comment:
seeing this now I'm surprised we didn't do backticks for any of them

Reply:
Single ones you mean?